### ISSUE TITLE
p2p: activate pex by default

### ISSUE THREAD

**Issue by ebuchman** [tendermint/tendermint#940]

**Comment by melekes** [comment]
```
I tried restarting my node with `--p2p.pex`


Zaki Manian [11:43] 
and it couldn’t sync


[11:43] 
restarted without it and syncing fast


[11:44] 
just a datapoint
```

**Comment by ebuchman** [comment]
Turns out this is breaking :)

**Comment by zramsay** [comment]
closed in #958
